src/clients/authorization/DetailCell.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+} from "@patternfly/react-core";
+
+import type ResourceServerRepresentation from "@keycloak/keycloak-admin-client/lib/defs/resourceServerRepresentation";
+import { useAdminClient, useFetch } from "../../context/auth/AdminClient";
+
+import "./detail-cell.css";
+
+type Scope = { id: string; name: string }[];
+
+type DetailCellProps = {
+  id: string;
+  clientId: string;
+  uris?: string[];
+};
+
+export const DetailCell = ({ id, clientId, uris }: DetailCellProps) => {
+  const { t } = useTranslation("clients");
+  const adminClient = useAdminClient();
+  const [scope, setScope] = useState<Scope>();
+  const [permissions, setPermissions] =
+    useState<ResourceServerRepresentation[]>();
+
+  useFetch(
+    async () => {
+      const scopes = await adminClient.clients.listScopesByResource({
+        id: clientId,
+        resourceName: id,
+      });
+      const permissions = await adminClient.clients.listPermissionsByResource({
+        id: clientId,
+        resourceId: id,
+      });
+      return { scopes, permissions };
+    },
+    ({ scopes, permissions }) => {
+      setScope(scopes);
+      setPermissions(permissions);
+    },
+    []
+  );
```

Comment:
I think these API calls can be ran in parallel:
```suggestion
  useFetch(
    () =>
      Promise.all([
        adminClient.clients.listScopesByResource({
          id: clientId,
          resourceName: id,
        }),
        adminClient.clients.listPermissionsByResource({
          id: clientId,
          resourceId: id,
        }),
      ]),
    ([scopes, permissions]) => {
      setScope(scopes);
      setPermissions(permissions);
    },
    []
  );
```